src/MangaStreamParser.ts

```diff
@@ -197,7 +196,7 @@ export class MangaStreamParser {
             }
 
             const title: string = $('a', obj).attr('title') ?? ''
-            const image = this.getImageSrc($('img', obj)) ?? ''
+            const image = this.getImageSrc($('img', obj), undefined) ?? ''
```

Comment:
What's the purpose for this and why should it be added? Has not been needed before?

Reply:
> Some mangastream sites use src attributes like "https://google.com/images/abc.png" and others '/images/abc.png".
I thought it might be a nice feature to have an already implemented system might we add some of these mangastream sources in the future.